### PR TITLE
remove extrapolation option

### PR DESCRIPTION
**Pull request by michaeljbrazell**
- removed extrapolation option, we never used it anyways and it's risky due to a possible negative viscosity
- removed density from gravity term
- should we remove the old wall model implementation?

### CODE REVIEW

**Review by sayerhs — APPROVED** [review]

**Comment by michaeljbrazell** [comment]
@gdeskos @shashankNREL what do you think? should I delete the old implementation of the wall model BCs or keep it around?

File: amr-wind/diffusion/incflo_diffusion.cpp [hunk 1]
```diff
@@ -133,16 +130,6 @@ void wall_model_bc(
...
-    // copies cell center to face
-    Real c0 = 1.0;
-    Real c1 = 0.0;
-
-    // linear extrapolate onto face
-    if (extrapolate) {
-        c0 = 1.5;
-        c1 = -0.5;
-    }
-
```
Comment (gdeskos) [inline amr-wind/diffusion/incflo_diffusion.cpp:145]: Don't we want to keep the "copy cell center to face" case ? This doesn't run the risk of negative viscosity. But I am happy to remove it all together if you don't think we are going to need it in the future

**Comment by shashankNREL** [comment]
> @gdeskos @shashankNREL what do you think? should I delete the old implementation of the wall model BCs or keep it around?

Please remove the old implementation of the wall model BCs.

File: amr-wind/diffusion/incflo_diffusion.cpp (hunk 1 above)
Comment (michaeljbrazell) [reply amr-wind/diffusion/incflo_diffusion.cpp:145]: that's what I did, the PR only does the copy from cell center to face. Unless I made a typo...

**Pull request merged**